### PR TITLE
[IMP] install: Install odoo requirements from original project

### PR DESCRIPTION
**Pull request by moylop260**
- Odoo now use a requirements.txt file for all versions.
 - Odoo requirements is different between versions (-+ packages, other versions)
 - Odoo requirements is updated frequently with important fixes.

We need use original requirements in order to avoid errors related with differences in packages version.

This PR helpful to avoid PRs to change requirements.txt like as:
 - https://github.com/OCA/maintainer-quality-tools/pull/446
 - https://github.com/OCA/maintainer-quality-tools/pull/402
 - https://github.com/OCA/maintainer-quality-tools/commit/03d026651500a4c01fb4aa45792c7f6f6e589625#diff-e5bd6ecd6853cdc5a0ff398dd0c55562
 - https://github.com/OCA/maintainer-quality-tools/commit/2ac0112759bd3d2df0122c07ec6305659ba41969#diff-e5bd6ecd6853cdc5a0ff398dd0c55562

The custom requirements packages was added from root requirements.txt file.
Detected from 
[diff-requirements-mqt-vx-odoo](https://docs.google.com/spreadsheets/d/1BuEjEhrdl2caXnl_YIH780hDAAqkP9oZREyZdoMebuU/edit#gid=0)

### CODE REVIEW

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -68,7 +68,11 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 pip install --upgrade pip
-pip install --user -q --no-binary pycparser -r $(dirname ${BASH_SOURCE[0]})/requirements.txt
+# Remove lxml from odoo requirements because is installed from apt package
+sed -i '/^lxml\=\=/d' ${ODOO_PATH}/requirements.txt
+# Remove python-ldap from odoo requirements because is not a common module used
+sed -i '/^python-ldap\=\=/d' ${ODOO_PATH}/requirements.txt
```
Comment (pedrobaeza) [inline travis/travis_install_nightly]: Is this one slow to be installed?

Comment (moylop260) [inline travis/travis_install_nightly]: `time pip install python-ldap==2.4.15`

```bash
real	0m5.831s
user	0m2.176s
sys	0m0.456s
```

That spent 5 seconds really is not too much but I'm using the same current behaviour where is currently is commented:
 - https://github.com/OCA/maintainer-quality-tools/blob/530d4c09e23b0de52d5cf01e222c16820b592fc3/travis/requirements.txt#L23

Comment (pedrobaeza) [inline travis/travis_install_nightly]: OK

**Comment by moylop260** [comment]
@lasley @Yajo @dreispt What do you think about?

**Comment by moylop260** [comment]
Thanks for review

@pedrobaeza 
Could you merge it, please?

File: travis/travis_install_nightly [hunk 2]
```diff
@@ -68,7 +68,16 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 pip install --upgrade pip
-pip install --user -q --no-binary pycparser -r $(dirname ${BASH_SOURCE[0]})/requirements.txt
```
Comment (pedrobaeza) [inline travis/travis_install_nightly]: @moylop260 This PR has broken l10n-spain: https://travis-ci.org/OCA/l10n-spain/jobs/250811101 and I think the problem is in this line.

**Pull request merged**